Misc/NEWS.d/next/Library/2019-09-06-17-40-34.bpo-37953.db5FQq.rst

```diff
@@ -0,0 +1,3 @@
+Forward references in the typing module now have different hash and eq methods. References will now compare equal properly if only one of them has been evaluated and the other hasn't. Hash values are now consistent across the lifetime of a forward reference. 

```

Comment:
Use `` `__hash__` `` and `` `__eq__` `` instead of just `hash` and `eq`. Also this news item is too long, couple short sentences (or even one) should be enough.